client/src/App.jsx

```diff
@@ -1,32 +1,63 @@
 import React from 'react';
 import axios from 'axios';
+import RelatedSongsList from './RelatedSongsList.jsx';
+import ArtistTooltip from './ArtistTooltip.jsx';
 
-class App extends React.Component { 
+class App extends React.Component {
     constructor(props) {
         super(props);
+        this.state = {
+            mainSong: [],
+            mainSongLikes: 0,
+            mainSongReposts: 0,
+            relatedSongs: [],
+            relatedPlaylist: []
+        };
 
-        this.getTest = this.getTest.bind(this);
+        this.getMainSong = this.getMainSong.bind(this);
     }
 
-    getTest () {
-        axios.get('/test')
+    getMainSong() {
+        return axios.get('/api/mainSong')
             .then(res => {
-                console.log('get test res ', res);
+                console.log('RES DATA FROM GET ALL SONGS GET REQUEST FROM CLIENT SIDE ', res.data);
+                this.setState({
+                    mainSong: res.data[0],
+                    mainSongLikes: res.data[0].likes,
+                    mainSongReposts: res.data[0].reposts,
+                    relatedSongs: res.data[0].related_songs,
+                    relatedPlaylist: res.data[0].related_playlists
+                })
+            })
+            .catch((err) => {
+                console.log('get err so many errs on client side ', err);
             })
-            .catch(err => console.log('get test err ', err))
     }
     
-    componentDidMount () {
-        this.getTest();
+    componentDidMount() {
+        this.getMainSong();
     }
-
+    
     render() {
         return (
+        <div className="content">
         <div>
-            <div>TESTING REACT</div>
-        </div>
-        )
-    }
+            <p><ArtistTooltip contents={<RelatedSongsList relatedSongs={this.state.relatedSongs} />} position={'bottom'}>Artist Name Test</ArtistTooltip></p>
```

Comment:
Is closing tag of ArtistTooltip necessary? I think you can just close the tag with a forward slash, unless it breaks the <p> tag... in which case, please ignore:
```suggestion
            <p><ArtistTooltip contents={<RelatedSongsList relatedSongs={this.state.relatedSongs} />} position={'bottom'} />Artist Name Test</p>
```

Reply:
actually it breaks up the p tag which is weird!